sql/Shard.go

```diff
@@ -483,7 +483,7 @@ func (s *scanError) Error() string {
 
 func structFromRow(outputItemStructVal reflect.Value, columns []string, rows *sql.Rows, query string, args []interface{}, log *log.Logger) errs.Err {
 	vals := make([]interface{}, len(columns))
-	for i, _ := range columns {
+	for i := range columns {
```

Comment:
Is this intentional?

Reply:
gofmt did that `gofmt -w -s ` a simplify bit